### ISSUE TITLE
Inaccurate hash rate calculation for some cryptocurrencies

### ISSUE THREAD

**Issue by jisaso** [FxPool/FXMinerProxy#8]
It has been observed that the hash rate calculations for some cryptocurrencies are not accurate. This may lead to misleading information or incorrect estimations for mining efficiency. Please investigate the issue and provide a solution to ensure precise calculations for all supported cryptocurrencies.

**Comment by FxPool** [comment]
Yes, this is because the difficulty of the mine pool changes so that the calculated hash is not accurate, we will try to fix this problem in the future version
